docs/shared/tutorials/angular-monorepo.md

```diff
@@ -1241,50 +1241,48 @@ This tutorial walked you through how Nx can improve the local development experi
 - Nx Atomizer [automatically splits](/ci/features/split-e2e-tasks) large e2e tests to distribute them across machines. Nx can also automatically [identify and rerun flaky e2e tests](/ci/features/flaky-tasks).
 
-### Generate a CI Workflow
-
-If you are starting a new project, you can use the following command to generate a CI workflow file.
+### Connect to Nx Cloud
 
-```shell
-npx nx generate ci-workflow --ci=github
-```
+Nx Cloud is a companion app for your CI system that provides remote caching, task distribution, e2e tests deflaking, better DX and more.
 
-{% callout type="note" title="Choose your CI provider" %}
-You can choose `github`, `circleci`, `azure`, `bitbucket-pipelines`, or `gitlab` for the `ci` flag.
+{% callout type="note" title="Commit and push your changes" %}
+Now that we're working on the CI pipeline, it is important for your changes to be pushed to a github repository. Commit all your changes so far and push them up to a hosted repository.
 {% /callout %}
 
-This generator creates a `.github/workflows/ci.yml` file that contains a CI pipeline that will run the `lint`, `test`, `build` and `e2e` tasks for projects that are affected by any given PR.
-
-The key line in the CI pipeline is:
+When we set up the repository at the beginning of this tutorial, we chose to use GitHub Actions as a CI provider. This created a basic CI pipeline and configured Nx Cloud in the repository. It also printed a url in the terminal to register your repository in your [Nx Cloud](https://cloud.nx.app) account. If you didn't click on the link when first creating your repository, you can show it again by running:
 
-```yml
-- run: npx nx affected -t lint test build e2e-ci
+```shell
+npx nx connect
 ```
 
-### Connect to Nx Cloud
+Once you click the link, follow the steps provided and make sure Nx Cloud is enabled on the main branch of your repository.
 
-Nx Cloud is a companion app for your CI system that provides remote caching, task distribution, e2e tests deflaking, better DX and more.
-
-To connect to Nx Cloud:
-
-- Commit and push your changes
-- Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
+### Configure Your CI Workflow
 
-#### Connect to Nx Cloud Manually
+When you chose GitHub Actions as your CI provider at the beginning of the tutorial, `create-nx-workspace` created a `.github/workflows/ci.yml` file that contains a CI pipeline that will run the `lint`, `test`, `build` and `e2e` tasks for projects that are affected by any given PR.
 
-If you are not able to connect via the automated process at [https://cloud.nx.app](https://cloud.nx.app), you can connect your workspace manually by running:
+If you need to generate the default pipeline, you can do so with this command:
 
 ```shell
-npx nx connect
+npx nx generate ci-workflow --ci github
```

Comment:
I think we can just say, if you need to generate a new workflow file for GitHub Actions or other providers, run `npx nx g ci-workflow`. The users can figure out the prompts, and we don't have to tell them the other values for `ci`.